azure_batch/main.tf

```
@@ -0,0 +1,183 @@
+/*
+
+# Purpose
+
+Create an Azure Batch account [1] with a pool populated of Docker-compatible compute nodes [2].
+
+# Notes
+
+1. Create a storage account [3] to store files used by the compute nodes to perform any backend 
+   calculation.
+2. Create a container within the above-mentioned storage account with a user-tunable retention 
+   policy of n days [4].
+3. Create a private container registry [5] and push a private Docker image in that registry.
+4. Create the Batch account. Warning: it is not possible to create more then a single Batch Account
+   per subscription and per region due to quota limitations (see Ref. [6]). Consider increasing that
+   if needed.
+5. Create a resizable pool (see Ref. [7] for more details about the autoscale formula) made of 
+   Docker-compatible nodes whose configuration is user-chosen [8]. Warning: by default, only a 
+   very limited array of VMs is available (10 VMs chosen amonst the Av2 Series, DSv3 Series, 
+   Dv3 Series, ESv3 Series and Ev3 Series). Should you need other types of VMs or more of them, 
+   consider asking for an increase of quota for your Azure subscription (see Ref. [6]).
```

Putting the warning on a new line as above:
```suggestion
5. Create a resizable pool with custom autoscaling formula [7] comprised of Docker-compatible
   nodes whose configuration is user-defined [8].
   Warning: by default, only a very limited array of VMs is available (10 VMs chosen amonst the
   Av2 Series, DSv3 Series, Dv3 Series, ESv3 Series and Ev3 Series). Should you need other
   types of VMs or more of them, consider asking for an increase of quota for your Azure
   subscription [6].
```